src/GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests/Services/ImporterLocationServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Services;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Database.StatisticsDbUtils;
+namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests.Services
+{
+    public class ImporterLocationServiceTests
+    {
+        [Fact]
+        public async Task Find_EnglishDevolvedAreaCaseSensitiveLocationName()
```

Comment:
Not sure if it's necessary but just pointing out this this test doesn't test this method with any pre-existing location data, only data which is added and saved through it not already existing. Might be a good idea to add an alternative to this with pre-existing data which should be found and returned without needing to be saved.